### PR TITLE
Update strings.xml

### PR DESCRIPTION
**Pull request by airon90**
Translation updated. If there are new keys please tag me.
Fixing part of #108

### CODE REVIEW

**Comment by airon90** [comment]
However, I think that some keys are missing ("User suggested apps", "Redditor made", ...)

File: app/src/main/res/values-it/strings.xml [hunk 1]
```diff
@@ -24,36 +24,38 @@
...
+        <item>Scuro</item>
         <item>AMOLED</item>
     </string-array>
     <string-array name="themesValues">
         <item>0</item>
         <item>1</item>
         <item>2</item>
     </string-array>
+ <string name="no_download_client">Nessun client disponibile per questo download</string>
+ <string name="submitapp">Suggerisci un'app</string>
```
Comment (d4rken) [inline app/src/main/res/values-it/strings.xml]: Need to escape apostrophes.

Comment (airon90) [reply app/src/main/res/values-it/strings.xml]: Fixed. Thank you

**Comment by bobheadxi** [comment]
@airon90 
> However, I think that some keys are missing ("User suggested apps", "Redditor made", ...)

I believe these (the categories) are not included because we pull them straight from the wiki, we don’t control what they’re called

**Pull request merged**